app/views/lessons/error.jbuilder

```diff
@@ -0,0 +1 @@
+json.deleted false
```

Comment:
Errors should be more meaningful and should be reused for all errors. Add error message here.